src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
Hmm, maybe we could restore that method then, because you could add a dependency on 2.x in your package.